### PR TITLE
Tests for Customer Tile

### PR DESCRIPTION
**Pull request by adityanaag3**
### What does this PR do?
Added tests for customerTile component

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### CODE REVIEW

File: es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            is: CustomerTile
+        });
+        element.customer = CUSTOMER_DETAILS;
+        element.object = SOBJECT_TYPE;
+        document.body.appendChild(element);
+
+        const lowerCaseObjectName = SOBJECT_TYPE.toLowerCase();
+
+        const iconElement = element.shadowRoot.querySelector('lightning-icon');
+        expect(iconElement.iconName).toBe(`standard:${lowerCaseObjectName}`);
```
Comment (pozil) [inline es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js:61]: Let's also check for `alternative-text` value here.

File: es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        document.body.appendChild(element);
+
+        return flushPromises().then(() => {
+            // Validate if event got fired
+            const linkEl = element.shadowRoot.querySelector('a');
+            expect(linkEl.getAttribute('alt')).toBe(
+                `Navigate to ${SOBJECT_TYPE} record detail for ${CUSTOMER_DETAILS.name}`
+            );
+            expect(linkEl.href).toBe(
+                `https://mydomain.my.salesforce.com/${CUSTOMER_DETAILS.Id}`
```
Comment (pozil) [inline es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js]: Move the static part of the URL as a module constant and add a comment to indicate that it's coming from the NavigationMixin stub.

**Review by pozil — CHANGES_REQUESTED** [review]

File: es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js (hunk 1 above)
Comment (adityanaag3) [reply es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js:61]: Done

File: es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js (hunk 2 above)
Comment (adityanaag3) [reply es-space-mgmt/main/default/lwc/customerTile/__tests__/customerTile.test.js]: Done

**Review by pozil — APPROVED** [review]

**Pull request merged**